app/views/apcsp/PageMarketing.vue

```diff
@@ -207,21 +207,25 @@
               .col-md-6
                 .question {{ $t('apcsp_marketing.question_3') }}
                 .answer(v-html="$t('apcsp_marketing.answer_3', i18nData)")
-    modal-apcsp-contact(v-if="showModal" @close="showModal = false")
+    modal-get-licenses(v-if="showRequestModal" @close="showRequestModal = false" :subtitle="'Let us know if you have any questions or requests regarding our AP CSP curriculum.'")
+    modal-apcsp-contact(v-if="showContactModal" @close="showContactModal = false")
```

Comment:
we dont want to use this modal in any case. Let's just use the modal-get-licenses only